src/coreclr/src/jit/emit.cpp

```diff
@@ -1121,6 +1121,34 @@ float emitter::insEvaluateExecutionCost(instrDesc* id)
     return max(throughput, latency);
 }
 
+//------------------------------------------------------------------------------------
+// perfScorepeUnhandledInstruction:
+//    Helper method used to report an unhandled instruction
+//
+// Arguments:
+//    id  - The current instruction descriptor to be evaluated
+//    pResult - pointer to struct holding the instruction characteristics
+//              updated with default one cycle latencies if we return
+//
+// Notes:
+//     When validating that the PerfScore handles every instruction.
+//     the #if 0 blcok is changed into a #ifdef DEBUG
+//     We will print the instruction and instruction grouyp
+//     and instead of returning we will assert
+//
+//     Otherwise we will return default latencies of 1 cycle.
+//
+void emitter::perfScoreUnhandledInstruction(instrDesc* id, insExecutionCharacteristics* pResult)
+{
+// Change this to #ifdef DEBUG to assert on any unhnadled instructions
```

Comment:
Typo (unhandled).
I'm not sure why you wouldn't leave this enabled in `DEBUG` builds, though I can see that it might be risky. That said, I don't know how often someone would think of re-enabling this if they were adding instructions.

Reply:
There is active work with adding instruction in the SIMD area.  As I will be out ofthe office shortly I didn't want to block them at this time.

If Tanner and others want I can enable this DEBUG assertion to enforce that they add latencies for any new instructions.

Reply:
It would be nice, IMO, to enforce this now while we are doing iterative work; rather than needing to go back and fixup everything later.

Reply:
I created PR #810 to turn this assert on